src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,34 @@
+.nav {
```

Comment:
nav should be inside header.scss

Reply:
Nope, since it is separate block, it's correct to put it in separate file.